wire/msgheaders.go

```diff
@@ -47,7 +48,7 @@ func (msg *MsgHeaders) BtcDecode(r io.Reader, pver uint32) error {
 	if count > MaxBlockHeadersPerMsg {
 		str := fmt.Sprintf("too many block headers for message "+
```

Comment:
Not a big deal, but I noticed you changed every other instance of this to `msg`.  Best to be consistent.